haiku-apps/bdirectconnect/bdirectconnect-0.5.0~git.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="A P2P file sharing client for the Direct Connect Protocol"
+DESCRIPTION="BeDC is a BeOS file sharing client compatible with the Direct \
+Connect protocol.
```

Comment:
I think we need a tiny change here, to mention the "long" name of the app. What do you think?
```diff
-DESCRIPTION="BeDC is a BeOS file sharing client compatible with the Direct \
-Connect protocol.
+DESCRIPTION="BDirectConnect (aka BeDC) is a BeOS file sharing client \
+compatible with the Direct Connect protocol.
```